### ISSUE TITLE
Warning and Critical Levels not working in 'updates' mode

### ISSUE THREAD

**Issue by geotekberlin** [nbuchwitz/check_pve#62]
In 'updates' mode the plugin returns correctly the number of linux updates pending, but it is impossible to set warning and critical levels for their numbers. Setting these levels doesn't have any effect. 

It would be desirable for the plugin to return a critical alert if a certain number of pending updates is exceeded.

**Comment by nbuchwitz** [comment]
That's correct. Until know I didn't see a need for this as one is usually interested to know "there are updates" and "everything is up to date". What what would be the benefit of a pure threshold based on the number of packages to update? Unfortunately this Proxmox API does not distinguish between security and other updates.

**Comment by geotekberlin** [comment]
Just one addition: The Icinga2-internal windows-update check allows for good reason an alert threshold for the number of pending updates that works well. I think it would be a good idea to make the PVE check consistent in this respect.
